airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/bulk/query.py

```diff
@@ -1502,3 +1504,483 @@ def record_process_components(self, record: MutableMapping[str, Any]) -> Optiona
                     transaction["order_id"] = record.get("id")
                     transaction["currency"] = record.get("currency")
                     yield self.process_transaction(transaction)
+
+
+class Product(ShopifyBulkQuery):
+    """
+    {
+        products(query: "updated_at:>='2020-01-20T00:00:00+00:00' AND updated_at:<'2024-04-25T00:00:00+00:00'", sortKey:UPDATED_AT) {
```

Comment:
@bazarnov Hi, out of curiosity, is this normal to have hard-coded updated_at values here?

Reply:
For the 'docstring' query example, I believe - yes. 

Does this look wrong to you? 
Should we have dynamic docstring for such cases?

If you can contribute to have the examples like this constructed dynamically, we would appreciate such improvements. 

Thank you for your question, and feel free to add more.